packages/muelu/src/Utils/MueLu_AggregationExportFactory_def.hpp

```diff
@@ -315,15 +306,6 @@ namespace MueLu {
         doJacksPlus_(vertices, geomSizes);
       else if(aggStyle == "Convex Hulls")
         doConvexHulls(vertices, geomSizes);
-      else if(aggStyle == "Alpha Hulls")
-      {
-        #ifdef HAVE_MUELU_CGAL
-        doAlphaHulls_(vertices, geomSizes);
-        #else
-        GetOStream(Warnings0) << "   Trilinos was not configured with CGAL so Alpha Hulls not available.\n   Using Convex Hulls instead." << std::endl;
-        doConvexHulls(vertices, geomSizes);
-        #endif
-      }
       else
       {
         GetOStream(Warnings0) << "   Unrecognized agg style.\n   Possible values are Point Cloud, Jacks, Jacks++, Convex Hulls and Alpha Hulls.\n   Defaulting to Point Cloud." << std::endl;
```

Comment:
"Alpha Hulls" is not longer valid, right?

Reply:
Good catch!